CHANGELOG.md

```diff
@@ -202,7 +203,7 @@ The changelog is available [on GitHub][2].
   Reexport `Bitraversable` related stuff from `base`.
 * [#100](https://github.com/kowainik/relude/issues/100):
   Add `Relude.Extra.Validation` with `Validation`data type.
-* [#89](https://github.com/kowainik/relude/issues/81):
+* [#89](https://github.com/kowainik/relude/issues/89):
```

Comment:
Thanks a lot for fixing old CHANGELOG entries! Very careful work that requires a lot of attention 👍